component-samples/demo/README.md

```diff
@@ -110,6 +110,15 @@ Note that this requires internet access by the component. Should a component be
 
 `ondie_check_revocations`: (optional, default = true for Manufacturer and Owner, false for RV) if "true" then revocations are checked by the component, no revocation checking is done if "false".
 
+To enable OnDie support in FDO PRI Manufacturer or FDO PRI Owner, update manufacturer.env or owner.env respectively as below.
```

Comment:
FDO PRI Manufacturer or FDO PRI Owner, update manufacturer.env or owner.env  -->

FDO PRI Manufacturer and FDO PRI Owner, update manufacturer.env and owner.env.

Reply:
Updated in 580a86295bf08d367b91f37c221e122ca651160d